src/10_functions.py

```diff
@@ -10,3 +11,9 @@
 
 # YOUR CODE HERE
 
+
+def is_even():
+    return 'Even!' if int(num) % 2 == 0 else 'Odd!'
+
```

Comment:
Good job on using a ternary to get this finished. I would just suggest that when you use a prefix with `is` or `has` in a function name, it should return a boolean rather than a string for other developers to understand it better.